Command/LoadDataFixturesDoctrineCommand.php

```diff
@@ -84,9 +90,29 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $em->getConnection()->connect($input->getOption('shard'));
         }
 
-        $fixtures = $this->fixturesLoader->getFixtures();
+        $dirOrFile = $input->getOption('fixtures');
+        if ($dirOrFile) {
+            $paths = is_array($dirOrFile) ? $dirOrFile : array($dirOrFile);
+        } else {
+            /** @var $kernel \Symfony\Component\HttpKernel\KernelInterface */
+            $kernel = $this->getApplication()->getKernel();
+            $paths = array($kernel->getRootDir().'/DataFixtures');
```

Comment:
What do you think if we use a separate variable for storing the string?

https://github.com/doctrine/DoctrineFixturesBundle/blob/old-rewrite/Service/LoaderService.php#L39

IMHO it's better to use `sprintf` to populate the value.
https://github.com/doctrine/DoctrineFixturesBundle/blob/old-rewrite/Service/LoaderService.php#L110